### ISSUE TITLE
[autoinstrumentation/apache] Incomplete description in Instrumentation CRD

### ISSUE THREAD

**Issue by bryan-aguilar** [open-telemetry/opentelemetry-operator#2198]
It appears that the instrumentation CRD  spec.apachehttpd.properties has an incomplete description. 

https://github.com/open-telemetry/opentelemetry-operator/blame/3b10d4cc4a64511c900343b13d5048408c592d40/config/crd/bases/opentelemetry.io_instrumentations.yaml#L58

```
apacheHttpd:
  description: ApacheHttpd defines configuration for Apache HTTPD auto-instrumentation.
  properties:
    attrs:
      description: 'Attrs defines Apache HTTPD agent specific attributes.
        The precedence is: `agent default attributes` > `instrument
        spec attributes` . Attributes are documented at https://github.'
```

**Comment by TylerHelmuth** [comment]
After doing some digging I find that this is happening because we limit the description length to 200 when generating the manifest.  I played around with that value and if we increase it then this description is fixed, but it also adds a lot of extra text to the other descriptions.

@open-telemetry/operator-maintainers how do you feel about increasing our description length?

**Comment by jaronoff97** [comment]
this is a tough balance to strike. I think @pavolloffay set this as an option recently because the CRD length was overflowing the max bytes. I think we should be showing everything in our API docs, but i don't think we can show everything in the CRD because of the overflow issue.

**Comment by jaronoff97** [comment]
🤦 [you're right](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#instrumentationspecapachehttpd) Can we run make api-docs off of non-truncated text? I think that would be an okay balance to strike.

**Comment by TylerHelmuth** [comment]
I played around with modifying the `api-docs` command to generate the crds in a temporary location but then ran into some problems with Kustomize.  I'm not very familiar with these tools, but I am pretty sure it is possible.

**Comment by jaronoff97** [comment]
Yeah this is def something we should look in to, im not sure what the lift will be. I can dig around a bit today.
